src/src/facilitator/facilitator.module.ts

```diff
@@ -38,52 +37,20 @@ import { FacilitatorService } from './facilitator.service';
 export class FacilitatorModule implements NestModule {
 	configure(consumer: MiddlewareConsumer) {
 		consumer.apply(AuthMiddleware).forRoutes('*');
-
-		consumer
-			.apply(ProgramMiddleware)
-			.exclude(
-				'/facilitators/getStatuswiseCount',
-				'/facilitators/forOrientation',
-				{
-					path: '/facilitators/experience/:id',
-					method: RequestMethod.DELETE,
-				},
-				{ path: '/facilitators/:id', method: RequestMethod.PATCH },
-				'/facilitators/program-facilitator/add',
-				'/facilitators/admin/okyc_details_override',
-				'/facilitators/admin/search-by-ids',
-				'/facilitators/admin/filter-by-beneficiaries',
-				'/facilitators/exportCsv',
-				'/facilitators/update-facilitator-aadhar/:id',
-				'/facilitators/admin/learner-status-distribution',
-				'/facilitators/',
-			)
-			.forRoutes(FacilitatorController);
-
-		consumer
-			.apply(AcademicYearIdMiddleware)
-			.exclude(
-				'/facilitators/getStatuswiseCount',
-				{
-					path: '/facilitators/experience/:id',
-					method: RequestMethod.DELETE,
-				},
-				{ path: '/facilitators/:id', method: RequestMethod.PATCH },
-				'/facilitators/admin/okyc_details_override',
-				'/facilitators/admin/search-by-ids',
-				'/facilitators/exportCsv',
-				'/facilitators/program-facilitator/add',
-				'/facilitators/admin/filter-by-beneficiaries',
-				{
-					path: '/facilitators/update-facilitator-aadhar/:id',
-					method: RequestMethod.PATCH,
-				},
-				'/facilitators/admin/learner-status-distribution',
-				{
-					path: '/facilitators/admin/prerak-learner-list/:id',
-					method: RequestMethod.GET,
-				},
-			)
-			.forRoutes(FacilitatorController);
+		consumer.apply(CohortMiddleware)
+		.exclude(
+			{
+				path: '/facilitators/experience/:id',
+				method: RequestMethod.DELETE,
+			},
+			{ path: '/facilitators/:id', method: RequestMethod.PATCH },
+			'/facilitators/program-facilitator/add',
```

Comment:
check if this is to be excluded '/facilitators/program-facilitator/add',

Reply:
resolved